packages/website/pages/account/payment.js

```diff
@@ -2,6 +2,7 @@
  * @fileoverview Account Payment Settings
  */
 
+import { parse as queryParse } from 'querystring';
```

Comment:
I think this will bundle in a browser polyfill of this thing from nodejs stdlib. wdyt of using the DOM URLSearchParams constructor? https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams

Reply:
@gobengo i looked and it looks like we're already bundling it.  i can follow up with a tech debt PR if you want to refactor all locations in the codebase to use `URL`.
![querystring](https://user-images.githubusercontent.com/2605980/196536662-c987b164-2362-43b0-98ca-a6ab19260844.png)